charts/covidDataExplorer/CovidExplorerTable.ts

```diff
@@ -258,12 +260,7 @@ export class CovidExplorerTable {
         return filtered.concat(continentRows, euRows)
     }
 
-    private static buildCovidVariableId(
-        name: MetricKind,
-        perCapita: number,
-        rollingAverage?: number,
-        daily?: boolean
-    ): number {
+    private static buildCovidVariableId(params: CovidQueryParams): number {
```

Comment:
Hmm, this makes it more difficult to override specific settings, and will make it a bit more difficult to build views (like the multimetric table) that differ from the URL.

Reply:
Will change URL Params to ExplorerOptions in a later commit.